.github/workflows/tests.yml

```diff
@@ -43,5 +43,11 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           python -m pip install tox tox-gh-actions
+      - name: Lint with tox
+        run: tox -e lint
+      # - name: Format with tox
+      #   run: tox -e format
```

Comment:
Would be great to make a separate PR with all the formatting changes so that version 0.5 is formatted

Reply:
Created this PR #448 . Would be great to merge before this.